src/components/DeployContract/DeployContract.tsx

```diff
@@ -28,6 +27,23 @@ function DeployContract(props: DeployContractProps) {
     setDevnetBaseUrlError(false);
   }
 
+  const requestAddTransaction = async (payload: any) => {
```

Comment:
@arayik-yervandyan  Also we have a addTransaction method in DeployScriptContent lets change this one too.